src/Test/Utilities/Portable/Assert/ConditionalFactAttribute.cs

```diff
@@ -260,16 +261,9 @@ public class WindowsDesktopOnly : ExecutionCondition
         public override string SkipReason => "Test only supported on Windows desktop";
     }
 
-    public class CovarantReturnRuntimeOnly : ExecutionCondition
+    public class CovariantReturnRuntimeOnly : ExecutionCondition
     {
-        public override bool ShouldSkip
-        {
-            get
-            {
-                // See if the runtime supports covariant returns.
-                return Type.GetType("System.Runtime.CompilerServices.RuntimeFeature")?.GetField("CovariantReturnsOfClasses") == null;
-            }
-        }
+        public override bool ShouldSkip => !ExecutionConditionUtil.RuntimeSupportsCovariantReturnsOfClasses;
```

Comment:
>ShouldSkip [](start = 29, length = 10)

Same comment.

Reply:
This is for conditioning tests that should only be run on runtimes that can handle covariant returns because their point is to verify the runtime behavior of the compiler-generated code.

---
In reply to: [461779636](https://github.com/dotnet/roslyn/pull/46367#discussion_r461779636) [](ancestors = 461779636)